src/main/kotlin/com/ekino/oss/errors/generator/ErrorBodyGenerator.kt

```diff
@@ -73,6 +73,11 @@ fun unprocessableEntity(service: String, code: String, devMessage: String, stack
   )
 }
 
+fun unsupportedMediaType(service: String, code: String, devMessage: String?, stacktrace: String): ErrorBody {
+  return toError(
+    service, HttpStatus.UNSUPPORTED_MEDIA_TYPE, code, HttpStatus.FORBIDDEN.reasonPhrase, devMessage, stacktrace, emptyList(), emptyList())
```

Comment:
Why don't you just use the `HttpStatus.UNSUPPORTED_MEDIA_TYPE.reasonPhrase` ("Unsupported Media Type") instead of the `HttpStatus.FORBIDDEN.reasonPhrase`?